tests/test_make_toy.py

```diff
@@ -0,0 +1,45 @@
+"""Integration tests for make_toy_data.py
+
+PREREQUISITES
+See https://github.com/broadinstitute/scp-ingest-pipeline#prerequisites
+
+Also, `cd tests` before running the examples below.
+
+EXAMPLES
+
+# Run all tests
+pytest
+
+# Run all tests and see print() output
+pytest -s
+
+# Run only tests in test_genomes.py
+pytest test_make_toy.py
+
+# Run all tests, show code coverage metrics
+pytest --cov=../ingest/
```

Comment:
same, I don't think it's useful to repeat pytest docs at the top of every test

Reply:
Agreed; details are in my reply over there.